tests/foreman/api/test_provisioningtemplate.py

```diff
@@ -26,60 +26,48 @@
 from fauxfactory import gen_mac
 from fauxfactory import gen_string
 from nailgun import client
-from nailgun import entities
 from requests.exceptions import HTTPError
 
-from robottelo.config import get_credentials
 from robottelo.config import settings
 from robottelo.config import user_nailgun_config
 from robottelo.utils.datafactory import invalid_names_list
 from robottelo.utils.datafactory import valid_data_list
 
 
-@pytest.fixture(scope="module")
-def module_location(module_location):
-    yield module_location
-    module_location.delete()
-
-
-@pytest.fixture(scope="module")
-def module_org(module_org):
-    yield module_org
-    module_org.delete()
-
-
-@pytest.fixture(scope="module")
-def module_user(module_org, module_location):
+@pytest.fixture(scope='module')
+def module_user(module_target_sat, module_org, module_location):
     """Creates an org admin role and user"""
     user_login = gen_string('alpha')
     user_password = gen_string('alpha')
-    # Create user with Manager role
-    orig_role = entities.Role().search(query={'search': 'name="Organization admin"'})[0]
-    new_role_dict = entities.Role(id=orig_role.id).clone(
+    orig_role = module_target_sat.api.Role().search(query={'search': 'name="Organization admin"'})[
+        0
+    ]
+    new_role_dict = module_target_sat.api.Role(id=orig_role.id).clone(
         data={
             'role': {
-                'name': f"test_template_admin_{gen_string('alphanumeric', 3)}",
+                'name': f'test_template_admin_{gen_string("alphanumeric", 3)}',
                 'organization_ids': [module_org.id],
                 'location_ids': [module_location.id],
             }
         }
     )
-    new_role = entities.Role(id=new_role_dict['id']).read()
-    user = entities.User(
+    new_role = module_target_sat.api.Role(id=new_role_dict['id']).read()
+    user = module_target_sat.api.User(
         role=[new_role],
         admin=False,
         login=user_login,
         password=user_password,
         organization=[module_org],
         location=[module_location],
     ).create()
-    yield (user, user_login, user_password)
+    user.password = user_password
+    yield user
     user.delete()
     new_role.delete()
 
 
-@pytest.fixture(scope="function")
-def tftpboot(module_org, target_sat):
+@pytest.fixture
+def tftpboot(module_org, module_target_sat):
```

Comment:
I think target_sat would be more appropriate here.

Reply:
no, I used module_target_sat as it is consistent across the module